src/Middleware/Diagnostics.Abstractions/src/IExceptionHandlerPathFeature.cs

```diff
@@ -12,6 +15,17 @@ public interface IExceptionHandlerPathFeature : IExceptionHandlerFeature
         /// The portion of the request path that identifies the requested resource. The value
         /// is un-escaped.
         /// </summary>
-        string? Path { get; }
+        new string? Path { get; }
```

Comment:
```suggestion
        new string Path => ((IExceptionHandlerFeature)this).Path;
```

Path shouldn't be null.